logins-sql/src/engine.rs

```diff
@@ -21,20 +22,20 @@ pub(crate) struct SyncInfo {
 // really a bundle of state that contains the sync storage client, the sync
 // state, and the login DB.
 pub struct PasswordEngine {
-    sync: Option<SyncInfo>,
+    sync: RefCell<Option<SyncInfo>>,
```

Comment:
Nit: We only ever use `replace` so we should use `Cell` instead of `RefCell` here.

Reply:
(If there's a reason we can't use `Cell`, `RefCell` is fine, but it has more overhead and is more complex than the fairly simple `Cell`)